src/App.tsx

```diff
@@ -64,15 +91,14 @@ function App() {
         placeholder="Enter github username"
         className="input-field"
         value={userName}
-        onChange={(e) => setUserName(e.target.value.trim())}
-        onKeyUp={() => handleKeyUp()}     
+        onChange={(e) => handleOnChange(e)}     
```

Comment:
```suggestion
        onChange={handleOnChange}
```
Can we write like this?